apps/site/src/content/events/consensus-toronto.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Consensus Toronto
+description: What is Consensus? Since 2015, Consensus has been more than an
+  event–it's the annual gathering of the tribes. It's where the most influential
+  voices in blockchain, Web3, and AI come together to drive innovation
+  forward.  In 2025, Consensus brings its flagship event to Toronto, where the
+  world's first Bitcoin and Ethereum ETFs were born. As North America's
+  third-largest tech ecosystem and second-largest financial hub, Toronto unites
+  global innovation with transformative technology leadership.  Built on
+  CoinDesk's foundation of independent, unbiased journalism, Consensus Toronto
+  will weave together breakthrough technologies, from digital assets to
+  artificial intelligence, fostering the collaborations that define what's
+  next.  Don’t miss this moment—join us in Toronto.
+created-on: 2025-02-12T15:30:00.000Z
+updated-on: ""
+published-on: ""
+category: supported-sponsored
+location:
+  primary: Toronto, Canada
+  region: north-america
+start-date: 2025-05-14T14:00:00.000Z
+end-date: 2025-05-16T11:00:00.000Z
+external-link: https://consensus2025.coindesk.com/
+seo:
```

Comment:
@CharlyMartin @YuliiaFilecoin - Is SEO description required? Is this one the reasons this is failing?